### ISSUE TITLE
Pain at the lateral aspect of the elbow

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10788]
**Preferred term label:**
Pain at the lateral aspect of the elbow
and a second term:
Pain at the posterior aspect of the elbow
and a third term
Pain at the medial aspect of the elbow

**Synonyms**
Lateral: Pain localized to the radial aspect of the elbow; lateral elbow pain; radial elbow pain
Posterior: Posterior elbow pain
Medial: Pain localized to the medial aspect of the elbow; medial elbow pain; ulnar elbow pain

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0030835

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Lateral: 
Radial tunnel syndrome
Radial head fracture
Capitellum fracture

Posterior:
Olecranon fracture

Medial:
Medial epicondylitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001173
- label: Pain at the lateral aspect of the elbow
- parents: [Elbow pain (HP:0030835)](https://hpo.jax.org/app/browse/term/HP:0030835)
- pmid: 
- def: Pain localized to the radial aspect of the elbow.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10788
- synonyms: Lateral elbow pain(NONE)|Radial elbow pain(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001174
- label: Pain at the posterior aspect of the elbow
- parents: [Elbow pain (HP:0030835)](https://hpo.jax.org/app/browse/term/HP:0030835)
- pmid: 
- def: Pain localized to the posterior aspect of the elbow.
- comment: The posterior aspect of the elbow refers to the back side of the elbow, which forms the prominent bony tip of the elbow.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10788
- synonyms:

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001175
- label: Pain at the medial aspect of the elbow
- parents: [Elbow pain (HP:0030835)](https://hpo.jax.org/app/browse/term/HP:0030835)
- pmid: 
- def: Pain localized to the medial aspect of the elbow.
- comment: The medial aspect of the elbow refers to the inner side of the elbow. Conditions characterized by this finding include medial epicondylitis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10788
- synonyms:
